napalm/nxos_ssh/nxos_ssh.py

```diff
@@ -1056,6 +1056,8 @@ def get_arp_table(self):
         for line in arp_entries.splitlines():
             if len(line.split()) == 4:
                 address, age, mac, interface = line.split()
+            elif len(line.split()) == 5:
+                address, age, mac, interface, flag = line.split()
```

Comment:
Does it make sense to test if the flag is one of the four options?

Reply:
I do not see the point in doing that personally unless we plan to use that value for something. Currently it is just assigned and later garbage collected.  But, if there is a use case for collecting it and using it in the code later I could do that and add appropriate testing. What did you have in mind?

Reply:
Or is your concern more that down the road some other value could land in there as Cisco changes the output some day?

Reply:
Yeah, something like:

```
address, age, mac, interface = line.split()[:4]
```

Might be more resilient.

Reply:
Fair enough!  I’ll change it here in a bit